etc/adminhtml/system/account.xml

```diff
@@ -206,6 +206,7 @@
                 <label>Debug email</label>
                 <comment><![CDATA[Debug information will be sent to these comma-separated email addresses.]]></comment>
                 <config_path>buckaroo_magento2/account/debug_email</config_path>
+                <backend_model>Buckaroo\Magento2\Model\Config\Backend\DebugEmails</backend_model>
```

Comment:
We can use also frontend validation:
`<validate>validate-emails</validate>`

With frontend validation I don't know if it is a must to have also backend validation for Admin configuration, is your choice